runtime/services/catalog/artifacts/artifacts_test.go

```diff
@@ -633,6 +633,61 @@ region: {{.env.region}}
 	}
 }
 
+func TestMetricsViewAvailableTimeZones(t *testing.T) {
+	repoStore := repoStore(t)
+	registryStore := registryStore(t)
+	tests := []struct {
+		name     string
+		filePath string
+		content  string
+		want     *drivers.CatalogEntry
```

Comment:
I would name `expected`

Reply:
Just copied it from the other test cases :)